src/dbt_junitxml/main.py

```diff
@@ -16,33 +20,97 @@ def convert_timestamp_to_isoformat(timestamp: str) -> str:
         '%Y-%m-%dT%H:%M:%S')
 
 
+def get_custom_properties(path: str, custom_properties: dict) -> dict:
```

Comment:
do you think it's hard to add unit tests here? cause it's quite hard to review it w/o unit tests

Reply:
add test, recreate pull request, close this one